source/blood/src/globals.cpp

```diff
@@ -79,11 +83,14 @@ void _consoleSysMsg(const char* pzFormat, ...) {
     OSD_Printf(OSDTEXT_RED "%s(%i): %s\n", _module, _line, buffer);
 }
 
-
 void __dassert(const char * pzExpr, const char * pzFile, int nLine)
 {
     LOG_F(ERROR, "Assertion failed: %s in file %s at line %i", pzExpr, pzFile, nLine);
 
+    // break into debugger, as one would expect from an assertion
+    // (unless NDEBUG is defined; this is handled in build/include/compat.h)
```

Comment:
See above

Reply:
I think at least the NDEBUG part is non-obvious